src/runtime/providers/ipx.ts

```diff
@@ -24,16 +24,17 @@ const operationsGenerator = createOperationsGenerator({
 })
 
 export const getImage: ProviderGetImage = (src, { modifiers = {}, baseURL = '/' } = {}) => {
-  const format = modifiers.format || '_'
-  delete modifiers.format
+  const _modifiers = JSON.parse(JSON.stringify(modifiers));
```

Comment:
- This issue might happen with other providers as well so needs to be done in src/runtime/image.ts ([https://github.com/nuxt/image/blob/0b10c22f1661e6013b1f364365e0e4dd04dfc423/src/runtime/image.ts#L98)
- I think one level modifiers reference copy should be enough

Reply:
> * This issue might happen with other providers as well so needs to be done in src/runtime/image.ts ([https://github.com/nuxt/image/blob/0b10c22f1661e6013b1f364365e0e4dd04dfc423/src/runtime/image.ts#L98)
> * I think one level modifiers reference copy should be enough

I fixed it.

* I moved logic to `https://github.com/nuxt/image/blob/0b10c22f1661e6013b1f364365e0e4dd04dfc423/src/runtime/image.ts#106` because it doesn't work if you do the same logic in line 98.